index.html

```diff
@@ -163,6 +163,38 @@ <h3>Production packages</h3>
     <h2>Who we work with</h2>
     <a name="Customers"></a>
   </div>
+  <div class="bg-accent-warm-light padding-4">
+    <div class="grid-row grid-gap">
+      <div class="tablet:grid-col-2"></div>
+      <div class="tablet:grid-col-2">
+        <img src="{{ site.baseurl }}/assets/images/logos/EPA-logo.svg" alt="EPA logo" class="square-10"/>
+      </div>
+      <div class="tablet:grid-col-8">
+        <h3>Environmental Protection Agency - AirNow.gov</h3>
+        <h4>EPA AirNow Application</h4>
+      </div>
+      <div class="tablet:grid-col-2"></div>
+    </div>
+    <div class="grid-row usa-prose">
+      <div class="tablet:grid-col-2"></div>
+        <div class="tablet:grid-col border-top border-base-light">
+        <p>
+          During the Canadian wildfires in the summer of 2023, traffic to <a href="https://www.airnow.gov">AirNow.gov</a> surged to several orders of magnitude higher than ever before. But thanks to the ability to easily scale applications on cloud.gov, the site remained up and completely healthy. 
+        </p>
+        <p>
+          “I will say this right up front – Cloud.gov is the single biggest factor in our success last week. The traffic we saw was astounding and beyond anything we tested for or ever even anticipated. Our ability to expand resources, combined with the CDN service in front of AirNow, made us incredibly stable.”
+        </p>
+        <p>
+          <a href="{{ site.baseurl }}/docs/customer-stories/epa-airnow-gov/" class="cg-arrow">Read the EPA AirNow customer story.</a>
```

Comment:
```suggestion
          <a href="{{ site.baseurl }}/docs/customer-stories/epa-airnow-gov/" class="cg-arrow">Read the EPA AirNow customer story</a>
```

Don't need the period in a link